python/mleap/sklearn/tree/tree.py

```diff
@@ -114,8 +114,8 @@ def serialize_to_bundle(self, transformer, path, model_name, serialize_node=True
         # Define attributes
         attributes = list()
         attributes.append(('num_features', transformer.n_features_))
-        if transformer.classes_ is not None:
-            attributes.append(('num_classes', transformer.n_classes_))
+        if transformer.n_classes_ is not None:
```

Comment:
we were using classes_ to determine when to append the num_classes to the attributes because that is not None when using a DecisionTreeClassifier but None when we have a DecisionTreeRegressor. perhaps it would be better to just check which type of transformer we're dealing with instead. what do you think?

Reply:
It would be way better to check the type of the classifier. classes_ handles a special case when the dimension of the output is greater than one (multi-output problem). I have never used it.

Reply:
sounds good, would it be ok to update the if statement then to check the type of the transformer?